README.md

```diff
@@ -10,12 +10,19 @@ This repository hosts the source code and releases of the new CLI.
 
 To install the CLI, use either [Homebrew](https://brew.sh/) (on Linux, macOS, or the Windows Subsystem for Linux) or [Scoop](https://scoop.sh/) (on Windows):
 
-### HomeBrew
+### Homebrew
 
 ```console
 brew install platformsh/tap/platformsh-cli
 ```
 
+After installing or updating platformsh-cli via Homebrew, make sure you have the right `platform` in your path by issuing
+```bash
+hash -r && ls -l $(which platform)
+```
+
+You should see a path that involves homebrew, `/opt/homebrew/bin/platform` on M1 mac, `/usr/local/Homebrew/bin/platform` on Intel mac, `/home/linuxbrew/.linuxbrew/bin/platform` on Linux. If you don't see that, then delete the target that you are seeing with `sudo rm $(which platform) && hash -r` and then you should see the correct `platform` tool in `which platform`.
```

Comment:
```suggestion
You should see a path that involves homebrew, `/opt/homebrew/bin/platform` on an Apple Silicon Mac (M1/M2), `/usr/local/bin/platform` on an Intel Mac, `/home/linuxbrew/.linuxbrew/bin/platform` on Linux. If you don't see that, then delete the target that you are seeing with `sudo rm $(which platform) && hash -r` and then you should see the correct `platform` tool in `which platform`.
```

Reply:
However, I'm not sure of what `hash -r` does

Reply:
`hash -r` updates the use of your $PATH. Otherwise, after removing something from your PATH you may get "/path/to/old/binary not found" after removing the item that *was* in the path.